File: Module - HTML Applications/HTML/app/src/main/java/com/javadude/html/JavaScriptGlue.java
package com.javadude.html;

import android.content.Context;
import android.webkit.JavascriptInterface;
import android.widget.Toast;

public class JavaScriptGlue {
	private Context context;

	public JavaScriptGlue(Context context) {
		this.context = context;
	}

	@JavascriptInterface
	public void toast(String message) {
		Toast.makeText(context, message, Toast.LENGTH_LONG).show();
	}
}

File: Module - HTML Applications/HTML2/app/src/main/java/com/javadude/html2/JavaScriptGlue.kt
package com.javadude.html2

import android.content.Context
import android.webkit.JavascriptInterface
import android.widget.Toast

class JavaScriptGlue(private val context: Context) {

    @JavascriptInterface
    fun toast(message: String) {
        Toast.makeText(context, message, Toast.LENGTH_LONG).show()
    }
}
